dev/Dockerfile

```diff
@@ -1,20 +1,8 @@
 FROM ubuntu:18.04
```

Comment:
I would name this file something else if have a similar one next to it with a dedicated name.

Just something to tell what its purpose is, unless you want to add a comment here instead

Reply:
👍🏽